server.go

```diff
@@ -1144,7 +1144,7 @@ func (s *Server) UnsubscribeClient(cl *Client) {
 		filters[i] = v
 		i++
 	}
-	s.hooks.OnUnsubscribed(cl, packets.Packet{Filters: filters})
+	s.hooks.OnUnsubscribed(cl, packets.Packet{FixedHeader: packets.FixedHeader{Type: packets.Unsubscribe}, Filters: filters})
```

Comment:
This feels unusual as there's an implicit assumption that any packet going to `OnUnsubscribed` would be an unsubscribe packet. However, I don't see anything fundamentally wrong with this, and it does make some sense. So I'm approving it 👍🏻